src/demos/_assets/locales.ts

```diff
@@ -0,0 +1,223 @@
+const locales = [
+  { name: "Afrikaans", locale: "af" },
+  { name: "Albanian", locale: "sq" },
```

Comment:
Please check with the localization team to confirm the list of locales we need to support. The `locales` list should reflect that (separate from this PR). I'd be useful to doc this list as well. cc @benelan

Reply:
https://github.com/Esri/calcite-components/issues/3660